### DISCUSSION TITLE
Welcome to react_todo Discussions!

### DISCUSSION THREAD

**Discussion by Julien7012** [Julien7012/react_todo#1] [Q&A] [answered]
how can I change this project into ts

**Comment by s0evil** [comment, marked as answer]
please send me your project url so I can convert it. Thank you.

**Comment by Julien7012** [comment]
Ok

**Comment by Julien7012** [comment]
How should I send this project via what?

**Comment by s0evil** [comment]
It's easy , please let me know your git repository url.

**Reply by Julien7012** [reply]
got it.
